h2/utilities.py

```diff
@@ -41,7 +41,7 @@
 ])
 
 
-def secure_headers(headers):
+def secure_headers(headers, hdr_validation_flags):
```

Comment:
This is a public function, although the only place it was used was in `stream.py`.

Is it a problem that the argument signature is changing? Or could we add an underscore and make it non-public, to match the other header pipeline functions in this file?